installer_scripts/install-yb-voyager

```diff
@@ -228,10 +280,166 @@ install_debezium_server() {
 	rm ${debezium_server_filename}
 }
 
+check_install_maven() {
+	MIN_REQUIRED_MAVEN_VERSION='3.8.4'
+	MAVEN_VER=$(mvn -version 2>&1 | grep "Apache Maven" | awk '{print $3}')
+    version_ok=$(version_satisfied "$MIN_REQUIRED_MAVEN_VERSION" "$MAVEN_VER")
+	if [[ $version_ok -eq 1 ]]
+	# if ([ -n "$MAVEN_VER" ] && (( 10#${MAVEN_VER} >= 10#${MIN_REQUIRED_MAVEN_VERSION} )) )
+        then 
+                output "Found sufficient maven version = ${MAVEN_VER}"
+        else
+                output "Maven not found or not sufficient. Installing..."
+                install_maven
+        fi
+}
+
+install_maven() {
+	# if dir already present, assuming it was installed in a previous run
+	if [ ! -d "/opt/yb-voyager/yb-debezium-maven-3.8.4" ]
+	then
+		sudo mkdir -p /opt/yb-voyager
+		MAVEN_URL="https://archive.apache.org/dist/maven/maven-3/3.8.4/binaries/apache-maven-3.8.4-bin.tar.gz"
+		wget -nv ${MAVEN_URL} -O apache-maven-3.8.4-bin.tar.gz 1>&2
+		tar -xzvf apache-maven-3.8.4-bin.tar.gz 1>&2
+		sudo mv apache-maven-3.8.4 yb-debezium-maven-3.8.4
+		sudo mv yb-debezium-maven-3.8.4 /opt/yb-voyager/
+		rm apache-maven-3.8.4-bin.tar.gz
+	fi
+	export PATH=/opt/yb-voyager/yb-debezium-maven-3.8.4/bin:$PATH
+}
+
+clean_debezium(){
+	pushd `git rev-parse --show-toplevel` > /dev/null
+	rm -rf debezium-server-voyager/target
+	popd > /dev/null
```

Comment:
How about saving the REPO_ROOT at the beginning of the script and then directly constructing abs paths in the commands:
`rm -rf $REPO_ROOT/debezium-server-voyager/target`

Reply:
Set REPO_ROOT=git rev-parse --show-toplevel
In cases where we just have to run some rm/mv commands, I've directly used REPO_ROOT. 
In cases where we need to be in the directory (to run mvn for example), I've used pushd `$REPO_ROOT/x/y/z.`